com.playeveryware.eos/Runtime/Core/Config/EOSConfig.cs

```diff
@@ -75,6 +75,13 @@ static EOSConfig()
         protected EOSConfig() : base("EpicOnlineServicesConfig.json")
         { }
 
+        protected override bool NeedsMigration()
+        {
+            // Because EOSConfig is becoming obsolete, it should never
+            // be "migrated", but left-as-is.
+            return false;
+        }
```

Comment:
A more verbose description of why this is the case would be a welcome addition to this function.